### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by JSCodes14**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
-    <title>To-Do List</title>
-    <link rel="stylesheet" href="css/styles.css" />
-  </head>
-
-  <body>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Task Master</title>
+    <link rel="stylesheet" href="styles.css">
```
Comment (andystoys) [inline src/index.html:7]: Can you keep the location of the style sheet the same as before? It does not live in the same directory as this file.

File: src/index.html [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
+        <h1>Task Master</h1>
+        <div class="task-input">
+            <input type="text" id="taskInput" placeholder="Add a new task...">
+            <button id="addTaskBtn">Add Task</button>
+        </div>
+        <ul id="taskList"></ul>
     </div>
-    <script src="js/script.js"></script>
-  </body>
+    <script src="script.js"></script>
```
Comment (andystoys) [inline src/index.html:18]: Can you keep the location of the js file the same as before? It does not live in the same directory as this file.

**Review by andystoys — CHANGES_REQUESTED** [review]
Can you keep the indentation the same as before for consistency?

**Review by Tenkaklet — CHANGES_REQUESTED** [review]
good stuff but need to follow directory structure.

**Pull request merged**